control_data_collecting_tool/scripts/data_collecting_trajectory_publisher.py

```diff
@@ -372,7 +711,7 @@ def updateNominalTargetTrajectory(self):
         lb = (l2 + l4) / 2
         if np.abs(la - lb) < 1e-6:
             la += 0.1  # long_side_length must not be equal to short_side_length
-        ld = np.sqrt(la**2 + lb**2)
+        # ld = np.sqrt(la**2 + lb**2)
```

Comment:
please remove commentout if it is not necessary.

Reply:
Thank you for your feedback. I removed commentout.